x-pack/metricbeat/module/aws/ec2/ec2.go

```diff
@@ -208,12 +222,18 @@ func createCloudWatchEvents(getMetricDataOutput *cloudwatch.GetMetricDataOutput,
 		mapOfMetricSetFieldResults[metricKey[0]] = fmt.Sprint(output.Values[0])
 	}
 
+	if len(mapOfMetricSetFieldResults) <= 3 {
+		errMsg := "Missing Cloudwatch data for instance " + instanceID + ". Please recheck the period setting in config."
```

Comment:
Should we log this every time or only once? Also I wonder on what level we should log it as it's bascially an expected error. For a new instances it's expected to happen.

Reply:
I'm logging this every time for each instance right now because this should only happen on new instance or when the period is set to a number that's not multiple of 300s/60s. But yes you are right, log level probably should be warning than error hmm

Reply:
Thanks, good to know!